templates/edge-computing/index.html

```diff
@@ -307,6 +310,17 @@ <h2>Integrate with the best hardware vendors</h2>
 </section>
 
 <section class="p-strip is-bordered">
+  <div class="u-fixed-width u-sv3 u-align--center u-hide--large u-hide--medium">
```

Comment:
We shouldn't do this, it's a convention across the site to hide images on smaller screens, see: https://ubuntu.com/openstack